Specification/2._Basic_concepts.md

```diff
@@ -67,17 +67,17 @@ allocation procedure.
 
 ## Service invocation
 
-Service invocation is a two-step data exchange between exactly two nodes: the client and the server.
+Service invocation is a two-step data exchange between exactly two nodes: a client and a server.
 Service invocation is performed in three steps:
 
 1. The client sends a *service request* to the server.
 2. The server application takes appropriate actions and returns the response data.
```

Comment:
Yes. I'll change it to a 2-step process. If we decide at some later point to cover different layers then we will need a diagram, and probably to cover the application and protocol layer behaviour on both nodes.